content/docs/gke/private-clusters.md

```diff
@@ -48,6 +48,11 @@ from the nodes. Google services (such as BigQuery and Cloud Storage) are still a
 Importantly this means you can continue to use your [Google Container Registry (GCR)](https://cloud.google.com/container-registry/docs/) to host your Docker images. Other Docker registries (for example, DockerHub) will not be accessible. If you need to use Docker images
 hosted outside GCR you can use the scripts provided by Kubeflow to mirror them to your GCR registry.
 
+There is however still a way to pull images directly from DockerHub, or other repositories, and not having the hassle of re-writing deployment files. Activating Google's [Cloud NAT](https://cloud.google.com/nat/docs/overview/)(Network Address Translation) service to the subnet you are using, will give your pods egress access.
+One use-case where this is very helpful is when you are using command tools that install several components and rely on publicly available images.
+[Istio](https://istio.io/docs/setup/install/istioctl/) with its istioctl is a good example: _istioctl manifest apply_
+Generating and editing the manifest file is timeconsuming and error-prone.
```

Comment:
Suggestion: Remove this sentence about editing the manifest file. As a reader, I found it a little unexpected at this location.